tests/conftest.py

```diff
@@ -279,11 +282,21 @@ def unprivileged_client(
             raise ClusterLoginError(user=non_admin_user_name)
 
 
+@pytest.fixture(scope="session")
+def dsci_resource(admin_client: DynamicClient) -> DSCInitialization:
+    name = py_config["dsci_name"]
+    for dsci in DSCInitialization.get(dyn_client=admin_client, name=name):
```

Comment:
If we already have the name, then why do we need to make get() call. We can create and use the python object here.

Reply:
done